### PR TITLE
Update the branch for the common-templates project

### PR DESCRIPTION
**Pull request by shwetaap**
Signed-off-by: Shweta Padubidri <spadubid@redhat.com>

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 periodics:
 - name: periodic-common-templates-nightly-fedora-cdi-builder
-  cron: "30 2 * * *"
+  cron: "30 14 * * *"
```
Comment (fgimenez) [inline github/ci/prow-deploy/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml:3]: Given that these tests require a bare metal it is better to execute them when there is less load in the system.

File: github/ci/prow-deploy/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@ periodics:
         requests:
           memory: "8Gi"
 - name: periodic-common-templates-nightly-centos-cdi-builder
-  cron: "30 3 * * *"
+  cron: "30 15 * * *"
```
Comment (fgimenez) [inline github/ci/prow-deploy/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml:45]: Same as above, please revert this value.

File: github/ci/prow-deploy/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml (hunk 1 above)
Comment (shwetaap) [reply github/ci/prow-deploy/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml]: Done

File: github/ci/prow-deploy/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml (hunk 2 above)
Comment (shwetaap) [reply github/ci/prow-deploy/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml]: Done

**Review by fgimenez — APPROVED** [review]
/lgtm
/approve

**Pull request merged**